src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { uuid } from 'uuid';
+
+const initialState = {
+  bookItems: [],
+};
+
+const bookSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.bookItems = [
+        ...state.bookItems,
+        {
+          id: uuid(),
+          title: action.payload.author,
+        },
```

Comment:
- [ ] I think the book state object should have `id`, `title` and `author`  properties but you are missing the `author` property. Also, you have mistakenly set the payload author value to the book title property so I suggest that you fix this to avoid issues with the book object. you can fix this by using the book state object below
```js
 {
      id: uuid(),
      title: action.payload.title,
      author: action.payload.author,
 },
```